src/wavm/mpi.cpp

```diff
@@ -168,6 +169,26 @@ WAVM_DEFINE_INTRINSIC_FUNCTION(env, "MPI_Init", I32, MPI_Init, I32 a, I32 b)
     return 0;
 }
 
+/**
+ * Returns the version of the standard corresponding to the current
+ * implementation.
+ *
+ * TODO not implemented.
+ */
+WAVM_DEFINE_INTRINSIC_FUNCTION(env,
+                               "MPI_Get_version",
+                               I32,
+                               MPI_Get_version,
+                               I32 version,
+                               I32 subversion)
+{
+    faabric::util::getLogger()->debug("S - MPI_Get_version");
```

Comment:
For debugging purposes I personally like to put the arguments in these logging statements. Not worth changing now, can just do in another PR.